README.md

```
@@ -20,12 +20,12 @@ Under the hood, MulmoClaude is an AI-native application platform: capabilities a
 
 > **[How AI-Native Applications Should Be Built](MANIFEST.md)** — the architecture, UX, and protocol thesis behind MulmoClaude.
 
-### 📣 Updates / アップデート情報
+### 📣 Updates
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)